packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFInstanceManager.h

```diff
@@ -61,10 +73,30 @@ NS_ASSUME_NONNULL_BEGIN
  * Retrieves the identifier paired with an instance.
  *
  * @param instance An instance that may be stored in the manager.
+ * @param willBePassed Whether the identifier will be passed to Dart. If YES, the strong reference
+ * to `instance` will be recreated and will need to be removed again by
+ * `removeStrongReferenceWithIdentifier:`.
+ *
+ * @return The  identifer associated with `instance` if the manager contains the value, otherwise
+ * NSNotFound.
+ */
+- (long)identifierForInstance:(nonnull NSObject *)instance
+    identifierWillBePassedToFlutter:(BOOL)willBePassed;
```

Comment:
My thoughts for the extra parameter was to prevent myself or a future maintainer from accidentally using this method only to check if the manager contains the instance. (and not use the result). The manager uses calls to this method to restore the strong reference to an instance and expects the Dart `InstanceManager` to restore the weak reference. I will update the documentation with the expectation of the `InstanceManager` when a method is called. 

I think I can also split these functionalities in to clearer method names. Thoughts on:

`identifierToBeReturnedToDartForInstance:`
`containsIdentifierForInstance:`

Reply:
The Dart methods could be:

`getInstanceReturnedFromHost(int identifier)`
`containsIdentifier(int identifier)`

Reply:
I went with `identifierWithStrongReferenceForInstance:` and `getInstanceWithWeakReference` with the latest patch.